README.md

```diff
@@ -3,3 +3,13 @@ File System Photo Upload Service
 ===================
 
 PO: Anthony James
+
+## Installation
+
+### MariaDB
+  1. `brew install mariadb`
+  1. `brew services start mariadb`
+
+## Redis
+  1. `brew install redis`
+  1. `brew services start redis`
```

Comment:
Do we need these if we have these services in our docker compose file?

Reply:
At this point, it's not working with redis/mysql containers. So a local server is needed